weather@mockturtl/files/weather@mockturtl/po/de.po

```diff
@@ -1,453 +1,402 @@
-msgstr "Einheit für die Temperatur:"
+#: applet.js:1340
+msgid "Clouds"
+msgstr "Wolken"
 
-#. cinnamon-weather->settings-schema.json->temperatureUnit->options
-msgid "fahrenheit"
-msgstr "Fahrenheit"
+#: applet.js:1340
+msgid "Mist"
+msgstr "Dunst"
 
-#. cinnamon-weather->settings-schema.json->temperatureUnit->options
-msgid "celsius"
-msgstr "Celsius"
+#: applet.js:1340
+msgid "Thunderstorm"
+msgstr "Gewitter"
 
-#. cinnamon-weather->settings-schema.json->translateCondition->description
-msgid "Translate condition"
-msgstr "Wetterbedingungen übersetzt anzeigen"
+#: applet.js:1340
+msgid "Rain"
+msgstr "Regen"
 
-#. cinnamon-weather->settings-schema.json->locationLabelOverride->description
-msgid "Override location label"
-msgstr "Ortsbezeichnung überschreiben mit:"
+#: applet.js:1340
+msgid "Snow"
+msgstr "Schnee"
 
-#. cinnamon-weather->settings-schema.json->show24Hours->description
-msgid "Display time in 24 hour format"
-msgstr "Zeit im 24-Stunden-Format anzeigen"
+#: applet.js:1340
+msgid "Drizzle"
+msgstr "Niesel"
 
-#. cinnamon-weather->settings-schema.json->forecastDays->description
-msgid "Forecast length"
-msgstr "Vorhersagezeitraum:"
+#: applet.js:1340
+msgid "Haze"
+msgstr "Dunst"
 
-#. cinnamon-weather->settings-schema.json->forecastDays->units
-msgid "days"
-msgstr "Tage"
+#: applet.js:1340
+msgid "Sleet"
+msgstr "Graupel"
 
-#. cinnamon-weather->settings-schema.json->showSunrise->description
-msgid "Show sunrise / sunset times"
-msgstr "Sonnenauf- und Sonnenuntergangszeit anzeigen"
+#: applet.js:1341
+msgid "Smoke"
+msgstr "Rauch"
+
+#: applet.js:1341
+msgid "Fog"
+msgstr "Nebel"
+
+#: applet.js:1341
+msgid "Sand"
+msgstr "Sand"
+
+#: applet.js:1341
+msgid "Dust"
+msgstr "Staub"
+
+#: applet.js:1341
+msgid "Sqalls"
+msgstr "Sturmböen"
+
+#: applet.js:1341
+msgid "Tornado"
+msgstr "Tornado"
+
+#: applet.js:1341
+msgid "Volcanic ash"
+msgstr "Asche"
+
+#: applet.js:1341
+msgid "Clear Sky"
+msgstr "Wolkenlos"
+
+#. settings-schema.json->head->description
+msgid "Settings for weather@mockturtl"
+msgstr "Einstellungen für „Wetter“"
+
+#. settings-schema.json->keybinding->description
+msgid "Set the keybinding to call the menu"
+msgstr "Tastenzuordnung für den Menüaufruf festlegen"
+
+#. settings-schema.json->providerSettings->description
+msgid "Weather Provider Settings"
+msgstr "Einstellungen für den Vorhersagedienst"
+
+#. settings-schema.json->dataService->description
+msgid "Data service"
+msgstr "Vorhersagedienst"
+
+#. settings-schema.json->dataService->options
+msgid "OpenWeatherMap"
+msgstr "OpenWeatherMap"
+
+#. settings-schema.json->apiKey->description
+msgid "API Key"
+msgstr "API-Schlüssel"
+
+#. settings-schema.json->apiKey->tooltip
+msgid "Copy this from your account on OpenWeatherMap and paste it here."
+msgstr "Kopiere ihn von Deinem Konto bei OpenWeatherMap und füge ihn hier ein."
+
+#. settings-schema.json->location->description
+msgid "Location"
+msgstr "Vorhersageort"
+
+#. settings-schema.json->location->tooltip
```

Comment:
@Gr3q This is the only thing I was unsure of since it differs from the rest. Since the source language works for the applet I just imitated the translation.

Reply:
I will have a look at it after work, but if assume it works for you